### ISSUE TITLE
Errors in set_value internals reference identifier that does not exist in scope

### ISSUE THREAD

**Issue by Adjective-Object** [jamalex/notion-py#14]
https://github.com/jamalex/notion-py/blob/master/notion/collection.py#L372

throws
```
  File "/home/adjective/.local/share/virtualenvs/notion-strip-7FiIVxN3/src/notion/notion/collection.py", line 373, in set_property
    path, val = self._convert_python_to_notion(val, prop)
  File "/home/adjective/.local/share/virtualenvs/notion-strip-7FiIVxN3/src/notion/notion/collection.py", line 392, in _convert_python_to_notion
    .format(val, identifier, valid_options))
NameError: name 'identifier' is not defined
```

**Comment by jamalex** [comment]
Fixed in https://github.com/jamalex/notion-py/pull/22, which will soon be released in 0.0.21

**Comment by jamalex** [comment]
Version 0.0.21 has now been [released to PyPI](https://pypi.org/project/notion/0.0.21/)! Please give it a try and see how it works for you. Thanks for reporting issues!
